content/correspondence/reference/API endpoints/confirmed/_index.nb.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Confirm
+linktitle: Confirm
+description: Endpoint for giving the correspondence status "Confirmed".
+
+weight: 60
+toc: true
+---
+
+## Endpoint
+
+POST /correspondence/api/v1/correspondence/{{correspondenceId}}/confirm
+
+## Description
+
+This endpoint sets the status of an existing correspondence to "Confirmed". Before using this endpoint, the status of the correspondence must be "Fetched". This status means that the correspondence has been downloaded. This occurs when a recipient uses the overview or details request. The status of a correspondence can be checked by using the overview endpoint described [here](https://docs.altinn.studio/api/correspondence/spec/#/Correspondence/get_correspondence_api_v1_correspondence__correspondenceId_).
```

Comment:
"This status means that the correspondence has been **retrieved by the recipient.**"

We have another endpoint called "Download" which I think the original sentence might confuse users with since it used the word download

Reply:
Godt innspill, jeg endrer på det!